docs/references/api/aggregate_functions.rst

```diff
@@ -0,0 +1,228 @@
+
+Using Aggregate Functions Within the Context of an Embedded Resource
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When embedding a resource, you can apply aggregate functions to columns from the associated resource to perform aggregations within the context of an embedded resource.
+
+Continuing with the example relationship between ``orders`` and ``customers`` from the previous section, imagine that you want to fetch the ``name``, ``city``, and ``state`` of all customers, along with the sum of their daily order amounts. This can be done in the following way:
+
+.. tabs::
+
+  .. code-tab:: http
+
+    GET /customers?select=name,city,state,orders(amount.sum(),order_date) HTTP/1.1
+
+  .. code-tab:: bash Curl
+
+    curl "http://localhost:3000/customers?select=name,city,state,orders(amount.sum(),order_date)"
+
+TODO: Add results.
+
+In this example, the ``amount`` column is summed and grouped by the ``order_date`` *within* the context of the embedded resource. That is, the ``name``, ``city``, and ``state`` from the ``customers`` table have no bearing on the aggregation performed in the context of the ``orders`` association; instead, the ``orders`` table is implicitly grouped by customer.
+
+Using Columns from a Spread Resource
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you :ref:`spread an embedded resource <spread_embed>`, the columns from the embedded resource that has been spread are treated as if they were columns of the top-level resource, both when using them as grouping columns and when applying aggregate functions to them.
+
+For instance, assume you want to sum the ``amount`` column from the ``orders`` table, using the ``city`` and ``state`` columns from the ``customers`` table as grouping columns. To achieve this, you may select these two columns from the ``customers`` table and spread them; they will then be used as grouping columns:
+
+.. tabs::
+
+  .. code-tab:: http
+
+    GET /orders?select=amount.sum(),...customers(city,state) HTTP/1.1
+
+  .. code-tab:: bash Curl
+
+    curl "http://localhost:3000/orders?select=amount.sum(),...customers(city,state)
+
+TODO: Add result.
+
+Now imagine that the ``customers`` table has a ``joined_date`` column that represents the date that the customer joined the platform. You want to get the oldest and youngest customer by join date for every distinct order date. This can be expressed as follows:
+
+.. tabs::
+
+  .. code-tab:: http
+
+    GET /orders?select=order_date,...customers(joined_date.max(),joined_date.min()) HTTP/1.1
+
+  .. code-tab:: bash Curl
+
+    curl "http://localhost:3000/orders?select=order_date,...customers(joined_date.max(),joined_date.min())
+
+As columns from a resource that has been spread are treated as if they were columns from the top-level resource, the ``max()`` and ``min()`` are applied *within* the context of the top-level, rather than within the context of the embedded resource, as in the previous section.
```

Comment:
Shortening as per the discussion above
```suggestion
As columns from a spreaded resource are treated as if they were columns from the top-level resource, the ``max()`` and ``min()`` are applied *within* the context of the top-level, rather than within the context of the embedded resource, as in the previous section.
```